news/12608-ability-to-set-default-env

```diff
@@ -0,0 +1,21 @@
+### Enhancements
+
+* Ability to set a default conda environment at launch, so when you open an Anaconda Prompt, or enter `conda activate` at the command line, it will activate your preferred environment rather than `base`. 
+* The command to set the default conda environment is `conda config --set default_start_environment my-favourite-env`.
+* If you don't set a default start environment, `conda activate` will continue to activate `base`.
+
+### Bug fixes
+
+* <news item>
+
+### Deprecations
+
+* <news item>
+
+### Docs
+
+* Updated `docs/user-guide/configuration/use-condarc.rst` to reflect this change.
```

Comment:
```suggestion
* Updated `docs/user-guide/configuration/use-condarc.rst` to describe the usage of a new configuration variable `default_start_environment`
```

When these items get complied into our release document, they will not be next to each other. So, it's better to be more specific when describing what is done. See this release document to get a better understanding of what I am talking about: https://github.com/conda/conda/releases/tag/23.3.0